controllers/articles.js

```diff
@@ -34,4 +34,73 @@ export default class ArticleController {
       return res.status(500).json({ success: false, error: [error.message] });
     }
   }
+
+  /**
+ * @description - Update an article
+ * @static
+ * @param {Object} req - the request object
+ * @param {Object} res - the response object
+ * @memberof ArticleController
+ * @returns {Object} class instance
+ */
+  static async updateArticle(req, res) {
+    const images = req.images || [];
+    const {
+      title, description, body
+    } = req.body;
+    const { id } = req.user;
+    const {
+      params: { slug }
+    } = req;
+    try {
+      const result = await Article.update({
+        title,
+        description,
+        body,
+        images,
+        userId: id
```

Comment:
@Dubby20 we don't need to update the `userId` since we've verified that the user is the author of the article.

Reply:
Alright